content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,14 +7,30 @@ menu:
     name: Release notes
 ---
 
+## v1.6.2 [2021-08-04]
+
+### Features
+
+- Add the `-template-id` option to `task list`.
```

Comment:
Does this refer to the `kapacitor flux task list`?
https://docs.influxdata.com/kapacitor/v1.6/working/flux/manage/list/

Reply:
Is this for `feat(tasks): add template-id to task list (#2621)` ? This is about the swagger API change to add data to the response, not a new option.

Reply:
Thank you @lesam!  I will swap the term "option" for "property".